pkgs/development/interpreters/ruby/default.nix

```diff
@@ -90,13 +90,18 @@ let
 
         enableParallelBuilding = true;
 
-        patches =
-          (import ./patchsets.nix {
-            inherit patchSet useRailsExpress ops fetchpatch;
-            patchLevel = ver.patchLevel;
-          }).${ver.majMinTiny}
-          ++ op (lib.versionOlder ver.majMin "3.1") ./do-not-regenerate-revision.h.patch
-          ++ op (atLeast30 && useBaseRuby) ./do-not-update-gems-baseruby.patch
+        patches = op (lib.versionOlder ver.majMin "3.1") ./do-not-regenerate-revision.h.patch
+          ++ op (atLeast30 && useBaseRuby) (
+            if atLeast32 then ./do-not-update-gems-baseruby-3.2.patch
```

Comment:
This needs to be dropped, `atLeast32` does not exist and we do not need it since we do not have Ruby 3.2 in 22.11.

Reply:
Thanks, that did the trick.